docs/scanner-pools.md

```diff
@@ -0,0 +1,135 @@
+
+- how much of the delegator or operator stake to be allocated,
+- how much commission you want from delegators' rewards.
+
+## Creating a new scanner pool
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click “Add Scanner Pool”
+![My Node Pools Tab](MyNodePoolsTab.png)
+
+- Select a chain for your node pool to monitor and register your node pool
+
+![Create Node Pool](CreateNodePool.png)
+
+- Accept the transaction for creating your node pool and wait for it to be confirmed
+
+## Depositing stake
+
+### Depositing on your own pool
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on node pool you would like to add stake
+
+![My Node Pools Tab](MyNodePoolsTab.png)
+
+- Click “Add Stake”
+
+![Add Stake](AddStakeToPool.png)
+
+- In order to stake there are 2 transactions that must be excecuted
+    - Approve the amount of FORT to stake
+    - Stake the approved FORT
+
+- Enter the amount of FORT you want to approve for staking and wait for the transaction to confirm
+
+![Add Stake](ApproveStaking.png)
+
+- Enter the amount of *approved* FORT you want to stake and wait for the transaction to confirm
+
+![Add Stake](ApproveFORTStaking.png)
+
+
+### Delegating to a pool
+
+- Find a node pool (that you do not own) on the Network Participants page that has a node and enough stake to earn rewards
+
+![Network Participants](NetworkParticipants.png)
+
+- Navigate to the node pool page by clicking on the address
+
+![Delegated Staking](DelegatedStaking.png)
+
+- Click on the “Delegate” button. There are 2 transactions required for delegation
+    - Approve the amount of FORT to delegate
+    - Delegate the approved FORT
+
+- Enter the amount of FORT you want to approve for delegation. Click “Approve tokens” and approve the transaction. Wait for the transaction to confirm.
+
+![Delegated Staking Approve](DelegatedStakingApprove.png)
+
+- Enter the amount of approved FORT want to delegate and click “Delegate” and approve the transaction. Wait for the transaction to confirm
+
+![Delegated Staking FORT](DSFORT.png)
+
+- Navigate to “My Stakes” or  “Overview” on your profile and you can see your staked FORT
+
+## Creating new nodes
+
+Each node needs to be registered to a specific pool in order to be operational. While one can register many scan nodes on a pool, the pool needs to be populated with more stake first in order to avoid registration issues. This is for preventing pool shutdown conditions that occur when the average per-scanner stake is below minimum.
+
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on the node pool you would like to add a node
+![My Node Pools List](MyNodePoolsList.png)
+
+- Make sure you have at least 2,500 FORT staked and click “Add Scan Node”. Use the sdk to generate the auth encoded key needed to claim a node into a pool
```

Comment:
@canercidam could you add details here on how someone generates the correct auth key that can be used to add a node to a node pool?